README.md

```diff
@@ -65,147 +65,227 @@ TechInfo lookup is looking at Toyota's Techinfo site (payment required, minimum
-    - Locate the necessary ECU codes.
-    - Add the ECU codes to `fingerprints.py`:
-      ```sh
-      nano /data/openpilot/selfdrive/car/toyota/fingerprints.py
-      ```
-    - Scroll down to the `CAR.TOYOTA_RAV4_PRIME` section and enter your corresponding ECU codes:
-      ```python
-      },
-      CAR.TOYOTA_RAV4_PRIME: {
-        (Ecu.engine, 0x700, None): [
-          b'\x01896634AJ7000\x00\x00\x00\x00',
-          b'\x018966342S7000\x00\x00\x00\x00',
-        ],
-        (Ecu.abs, 0x7b0, None): [
-          b'\x01F15264284100\x00\x00\x00\x00',
-          b'\x01F15264228300\x00\x00\x00\x00',
-        ],
-        (Ecu.eps, 0x7a1, None): [
-          b'\x018965B4233100\x00\x00\x00\x00',
-          b'\x018965B4209000\x00\x00\x00\x00',
-        ],
-        (Ecu.fwdRadar, 0x750, 0xf): [
-          b'\x018821F6201300\x00\x00\x00\x00',
-          b'\x018821F3301400\x00\x00\x00\x00',
-        ],
-        (Ecu.fwdCamera, 0x750, 0x6d): [
-          b'\x028646F4210100\x00\x00\x00\x008646G3305000\x00\x00\x00\x00',
-          b'\x028646F4205200\x00\x00\x00\x008646G4202000\x00\x00\x00\x00',
-        ],
-      ```
-
-12. Disable updates
-
-    ```sh
-    echo -en "1" > /data/params/d/DisableUpdates
-    ```
+1-3. Also connect the OBD2 connector to make sure the Comma stays powered on while turning the car on and off. You can experiment with not using OBD2 once everything's working, but use it for now.
+
+1-4. Turn on your Comma 3X by plugging in the right-angled USB-C cable to the harness.
+
+1-5. Connect it to your Wi-Fi network.
+
+1-6. Install "Custom Software" with URL `commaai/master`
+
+### Step 2. SSH into the device
+2-1. Before you start (for both macOS and Windows): https://github.com/commaai/openpilot/wiki/SSH#before-you-start
+
+2-2-1. macOS: https://github.com/commaai/openpilot/wiki/SSH#option-2mac---pre-installed-openssh-client-on-macos
+
+2-2-2. Windows: https://github.com/commaai/openpilot/wiki/SSH#option-2---pre-installed-openssh-client-on-windows-10-and-up
+
+2-3. SSH into the device.
+```sh
+ssh comma@"your Comma IP"
+```
+
+### Step 3. Extract security key
+> [!TIP]
+> If you already have the security key, proceed to Step 5. 
 
-12. Reboot the device:
-    - Enter the reboot command:
-      ```sh
-      sudo reboot
-      ```
+3-1. Put the car into `Not Ready to Drive` mode.
+* Slowly press the `POWER` button twice WITHOUT pressing the brake pedal.
+* The first press turns on `ACCESSORY` mode. The second press activates `Not Ready To Drive` mode shown below.
+  ![PXL_20240718_234619671 MP](https://github.com/user-attachments/assets/4970e82e-e7df-471f-9896-ba532509793d)
 
-13. Now, you should be ready to go!!!
+3-2. Kill the openpilot process.
+```sh
+pkill -f openpilot
+```
+Comma should display just the splash screen with the Comma logo.
 
+3-3. Clone Willem's secoc Git repository.
+```sh
+git clone https://github.com/I-CAN-hack/secoc
+```
+
+3-4. Navigate to the secoc directory.
+```sh
+cd /data/openpilot/secoc
```

Comment:
Changed to absolute path because one time SSH dropped me into `~comma` instead of `/data/openpilot/`

Reply:
ok